ntc_templates/templates/hp_comware_display_interface.textfsm

```diff
@@ -180,7 +180,7 @@ Start
   ^\s*GRE
   ^\s*Checksumming\sof\sGRE
   ^\s+-\s+ignored
-  ^\s+-\s+frame
+  ^\s+[-\d]+\s+frame
```

Comment:
I don't believe we want `-1` or `1-`.

I believe the below regex is what we're looking for.

```suggestion
  ^\s+(-|\d+)\s+frame
```